src/Message.php

```diff
@@ -16,7 +16,7 @@
     public function __construct($payload, $timestamp, $encoding)
     {
         $this->payload = $payload;
-        $this->timestamp = new \DateTimeImmutable($timestamp);
+        $this->timestamp = new \DateTimeImmutable("@$timestamp");
```

Comment:
`@` is not uded as for error suppression in here, it something like `at` 10 o'clock, I'm getting an Exception when I try to use the UNIX timestamp as a string or even integer.
I found that in here: http://php.net/manual/en/datetime.construct.php#example-2491
and this comment explains it: http://php.net/manual/en/datetime.construct.php#106979